srcs/parser/parser.cpp

```diff
@@ -42,7 +42,7 @@ bool	is_good_port(std::string port)
 {
 	int port_i;
 
-	port_i = atoi(port.c_str());
+	port_i = atoi(port.c_str());		// Doubt 1
```

Comment:
This one makes sense, takes the port as a string, transform into int and check if its inside the range of valid ports